src/data_forwarding/segment/forwarder.py

```diff
@@ -0,0 +1,133 @@
+from __future__ import annotations
+
+import logging
+from typing import Any, ClassVar
+
+from data_forwarding.base import BaseDataForwarder
+from sentry import VERSION, http
+from sentry.integrations.models.data_forwarder_project import DataForwarderProject
+from sentry.integrations.types import DataForwarderProviderSlug
+from sentry.services.eventstore.models import Event
+
+logger = logging.getLogger(__name__)
+
+DESCRIPTION = """
+Send Sentry events to Segment.
+
+This integration allows you to forward Sentry error events to Segment for unified analytics.
+
+Segment is a customer data platform (CDP) that helps you collect, clean, and control your customer data.
+"""
+
+
+class SegmentForwarder(BaseDataForwarder):
+    provider = DataForwarderProviderSlug.SEGMENT
+    rate_limit = (200, 1)
+    description = DESCRIPTION
+    endpoint: ClassVar[str] = "https://api.segment.io/v1/track"
+
+    @classmethod
+    def validate_event(cls, event: Event) -> bool:
```

Comment:
er could we get some comments on the why of each check? LIke we only support error events? What does the user_interface part/check mean?